### ISSUE TITLE
Build instructions incomplete

### ISSUE THREAD

**Issue by jherico** [nvpro-pipeline/VkHLF#5]
The sample solution appears to depend on corresponding locally created builds of the VulkanSDK glslang and spirv-tools projects, but they are not mentioned in the build instructions.

**Comment by Delwin9999** [comment]
Looking at the CMakeLists.txt it's looking for the environment variable VK_SDK_PATH which I assume is installed with the LunarG Vulkan SDK.  

We all should really get together and make a solid FindVulkan.cmake and submit for inclusion in CMake.  It would make all of this a lot easier.

**Comment by jherico** [comment]
CMake 3.7 already includes a native finder for Vulkan.  The issue here is that the Vulkan SDK includes libraries for Vulkan, but none of the SPIRV tool executables or libraries and not the glslang library (which is required if you want to compile shaders to SPIRV at runtime).  When I ran my initial CMake on this project, it worked fine, but when I actually ran the build it complained it couldn't find things like glslang.lib in `${VK_SDK_PATH}/glslang/build`.  I was able to fix that by running cmake / builds in both `${VK_SDK_PATH}/glslang/build` and `${VK_SDK_PATH}/spirv-tools/build` and then re-building VkHLF.  

The build instructions should either include instructions on how to pre-build the required glslang and spirv-tools requirements, or ideally, automatically build them as external projects.
